src/environment/mod_test.rs

```diff
@@ -477,6 +478,27 @@ fn setup_env_for_git_repo_with_values() {
     }
 }
 
+#[test]
+fn setup_env_for_execution_ci_check() {
+    env::remove_var("CARGO_MAKE_IS_CI");
```

Comment:
kinda scary, who knows if it will hurt later tasks in the flow if those are removed.
need to rethink this test somehow.